chart/templates/vmagent/deployment.yaml

```diff
@@ -84,12 +84,7 @@ spec:
           - --path.sysfs=/host/sys
           - --path.rootfs=/host/root
           - --no-collector.wifi
```

Comment:
could you put a comment explaining that collectors are in sync with alerting rules?